docs/content/standards/closed-loop-token.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Closed Loop Token
+---
+
+import BalanceTokenCoin from '@site/static/img/balance-token-coin.png';
+
+Closed Loop Token allows its creator to limit applications where the token is used, set up custom policies for transfers, spending and conversions. It is defined in the `sui::token` module and is a part of the Sui Framework.
+
+## Background & Use Cases
+
+The standard Coin implementation on Sui is an example of an open-loop system - Coins are free-flowing, [wrappable](/concepts/object-ownership/wrapped), [freely transferable](/concepts/dynamic-fields/transfers/custom-rules#the-store-ability-and-transfer-rules) and can be stored in any application. The best real world analogy would be "cash" - hardly regulated and can be freely used and passed.
+
+However, some applications require constraining the scope of the token to a specific purpose. For example, a token that can only be used for a specific service, or that can be used only by an authorized account, or one whose accounts can be blocked. A real-world analogy would be a "bank account" - regulated, controlled by the bank and compliant with certain rules and policies.
```

Comment:
I have to say that the bank account reference does not resonate with me. But it's all good. Just mentioning in case you want to remove it. I think the 3 examples below are enough and good for me